src/interceptors/ClientRequest/MockHttpSocket.ts

```diff
@@ -374,6 +374,7 @@ export class MockHttpSocket extends MockSocket {
       credentials: 'same-origin',
       // @ts-expect-error Undocumented Fetch property.
       duplex: canHaveBody ? 'half' : undefined,
+      // @ts-ignore
```

Comment:
This isn't something we should ignore. We likely have the wrong @types/node in the project so `Readable.toWeb()` is not present. The fix would be to find that type, not to add ts-ignore.

Reply:
You can also ignore these errors for now since they aren't directly related to the bugfix!

Reply:
I just had the wrong version of node types. I reinstall deps.